book/src/concepts/generated_qobject.md

```diff
@@ -7,100 +7,39 @@ SPDX-License-Identifier: MIT OR Apache-2.0
 
 # The generated QObject
 
-<!--
-TODO: this page needs rewriting
--->
-
 One of the key features of CXX-Qt is the ability to create your own QObjects from Rust.
 This is what the [`#[qobject]` macro](../bridge/extern_rustqt.md#qobjects) is for.
 This page serves to document the details of what is generated and how to interact with the generated QObject from Rust.
 
-The `#[qobject]` macro generates a QObject for a given Rust struct.
+The `#[qobject]` macro generates a QObject for a type alias to a Rust struct.
 Whilst this QObject is a C++ type, CXX-Qt will automatically wrap it as a [CXX Opaque Type](https://cxx.rs/extern-c++.html#opaque-c-types).
-These generated QObjects are accessible to Rust in a generated module with the name `qobject`. Each struct `T`'s generated QObject is accessible as `qobject::T`.
+
+> If the bridge module is named `qobject`, then the C++ type can be reached via `qobject::T`
 
 ## Anatomy
 
 Any QObject generated by CXX-Qt is just a C++ QObject subclass that owns an instance of the Rust struct.
 The instance of the Rust struct is constructed using its [`Default`](https://doc.rust-lang.org/std/default/trait.Default.html) implementation in the C++ constructor.
 Therefore implementing `Default` on the Rust struct is currently mandatory.
```

Comment:
This is no longer true. You can use `Constructor` instead.

Reply:
the next comment was the change to fix this I assume ?

Reply:
Yup, sorry for the duplicate.